code/__DEFINES/admin.dm

```diff
@@ -73,5 +73,8 @@
 #define SPAM_TRIGGER_WARNING	5	//Number of identical messages required before the spam-prevention will warn you to stfu
 #define SPAM_TRIGGER_AUTOMUTE	10	//Number of identical messages required before the spam-prevention will automute you
 
+#define MAX_KEYPRESS_COMMANDLENGTH 32 //Max length of a keypress command before it's considered to be a forged packet
+#define MAX_KEYPRESS_AUTOKICK

```

Comment:
Empty define?

Reply:
defined later I believe, or in config.

Reply:
No it's not defined later. Please actually check things before dismissing them